### PR TITLE
coinjoin whitepaper series

### PR DESCRIPTION
**Pull request by btsea**
ready to go from my end.

### CODE REVIEW

File: _posts/2020-02-19-whitepaper-series-11-coinjoin.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Attacks, Mitigations
+- [CoinJoin Sudoku](https://www.coinjoinsudoku.com/)
+- [The price of anonymity: empirical evidence from a market for Bitcoin anonymization](https://academic.oup.com/cybersecurity/article/3/2/127/4057584)
+- [When the cookie meets the blockchain: Privacy risks of web payments via cryptocurrencie](https://arxiv.org/pdf/1708.04748.pdf)
+- [Towards Characterizing Sybil Attacks in Cryptocurrency Mixers](https://eprint.iacr.org/2019/1111.pdf)
+- [Non-equal value CoinJoins. Opinion](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-December/017541.html)
+- [Sybil-Resistant Mixing for Bitcoin](https://people.cs.umass.edu/~gbiss/mixing.pdf)
+- [Fidelity bonds](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-July/017169.html)
+
+Other topics we will cover
```
Comment (jnewbery) [inline _posts/2020-02-19-whitepaper-series-11-coinjoin.md]:
```suggestion
## Other topics we will cover
```

File: _posts/2020-02-19-whitepaper-series-11-coinjoin.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Post-mix merge avoidance
+- Hardware wallets and conjoin 
+- How schnorr can improve coinjoins
```
Comment (jnewbery) [inline _posts/2020-02-19-whitepaper-series-11-coinjoin.md]:
```suggestion
- How schnorr signatures can improve CoinJoins
```

File: _posts/2020-02-19-whitepaper-series-11-coinjoin.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- [The price of anonymity: empirical evidence from a market for Bitcoin anonymization](https://academic.oup.com/cybersecurity/article/3/2/127/4057584)
+- [When the cookie meets the blockchain: Privacy risks of web payments via cryptocurrencie](https://arxiv.org/pdf/1708.04748.pdf)
+- [Towards Characterizing Sybil Attacks in Cryptocurrency Mixers](https://eprint.iacr.org/2019/1111.pdf)
+- [Non-equal value CoinJoins. Opinion](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-December/017541.html)
+- [Sybil-Resistant Mixing for Bitcoin](https://people.cs.umass.edu/~gbiss/mixing.pdf)
+- [Fidelity bonds](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-July/017169.html)
+
+Other topics we will cover
+- Post-mix merge avoidance
+- Hardware wallets and conjoin
```
Comment (jnewbery) [inline _posts/2020-02-19-whitepaper-series-11-coinjoin.md]:
```suggestion
- Hardware wallets and CoinJoin 
```

File: _posts/2020-02-19-whitepaper-series-11-coinjoin.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  - Whirlpool
+  - Wasabi
+- [Maker-Taker](https://github.com/JoinMarket-Org/JoinMarket-Docs/blob/master/High-level-design.md#maker)
+  - Joinmarket
+- [Coinshuffle](https://petsymposium.org/2014/papers/Ruffing.pdf)
+  - [Presentation](https://pdfs.semanticscholar.org/1ed4/b1b4e2b7fd154dc3060b2382d4de79ea3994.pdf)
+- [Coinshuffle++](https://www.ndss-symposium.org/wp-content/uploads/2017/09/ndss201701-4RuffingPaper.pdf)
+  - [Decred](https://cspp.decred.org/)
+- [ValueShuffle](https://eprint.iacr.org/2017/238.pdf)
+- [DiceMix](https://eprint.iacr.org/2016/824)
```
[the remark this answers is not in the document]
Comment (btsea) [reply _posts/2020-02-19-whitepaper-series-11-coinjoin.md]: Well they have a live implementation, so I wanted to highlight that. Perhaps there is a better way for me to do that.

**Comment by jnewbery** [comment]
Thanks @btsea 

Going forwards, I think it's better to squash commits before merging, to keep the git history cleaner.

**Pull request merged**